recipes/pytorch-dp/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytorch-dp" %}
+{% set version = "0.1b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65452d4754f76652030f3f847e29af72d87435142ee6f628d75e53c728067371
+
+build:
+  number: 0
+  skip: True  # [win or not py37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - torchvision
```

Comment:
there has been ALOT of talk of removing the pytorch stack on conda-forge.

We:
1. Don't have enough time to compile it all on CIs
2. Have yet to compile it for use with GPUs.

Maybe you could help us update the pytorch stack? or remove this dependency?

Reply:
I'd be happy to help where I can. But I don't think we can remove torchvision as a dependency.

Reply:
Is torchvision needed to build the package?

Reply:
It is unlikely torchvision is needed as to build